Tests iOS/MyList/SearchTests.swift

```diff
@@ -178,4 +178,28 @@ class SearchTests: XCTestCase {
         }
         app.saves.filterButton(for: "Search").wait().tap()
     }
+
+    // MARK: - Recent Search
+    func test_submitSearch_fromRecentSearch() {
+        app.launch()
+        tapSearch()
+        let searchField = app.navigationBar.searchFields["Search"].wait()
+        searchField.tap()
+        searchField.typeText("item\n")
+        var searchView = app.saves.searchView.searchResultsView.wait()
+        XCTAssertEqual(searchView.cells.count, 2)
+
+        app.navigationBar.buttons["Cancel"].tap()
+
+        searchField.tap()
+        searchField.typeText("test\n")
+        searchView = app.saves.searchView.searchResultsView.wait()
+
+        app.navigationBar.buttons["Cancel"].tap()
+
+        let app = XCUIApplication()
+        searchField.tap()
+        app.collectionViews["recent-searches"].staticTexts["item"].tap()
```

Comment:
unfamiliar with XCTest and how to get the test to tap the list item `item`.   If there is a better solution, please let me know.

Reply:
this seems fine to me, can we use `app.saves.searchView.recentSearchesView` instead of `collectionViews["recent-searches"]`. If you want to get fancy, you can also make a element (similar to when we use `app.saves.itemView(matching: "Item 1")`) so that you can use 
`app.saves.searchView.recentSearchesView(matching: "item")`